docs/src/ScrollableContainer.doc.js

```diff
@@ -591,4 +600,19 @@ function ScrollableContainerExample() {
   </MainSection>,
 );
 
+card(
+  <MainSection
+    name="Related"
+    description={`
+[Modal](/Modal) / [Sheet](/Sheet)
+
+Modal and Sheet components have built ScrollableContainer internally to support the correct positioning of children on scroll as well as to keep the header and footer shadows when scrolling the content. Do not pass children to Modal and Sheet wrapped in ScrollableContainer so that they don’t loose the scrolling shadows.
+
+[Tooltip](/Tooltip) / [Flyout](/Flyout) / [Typeahead](/Typeahead) / [Dropdown](/Dropdown)
+
+These are anchored components. When they are anchored to elements located within a scrolling container, they require ScrollableContainer for proper positioning and ensuring they remained attached to its anchor when scrolling. If they are located within scrolling Modal and Sheet components, ScrollableContainer isn't needed as ScrollableContainer is already built-in.
```

Comment:
"ScrollableContainer must be used around any of these components if they are used within a container that could possibly scroll. This is necessary to ensure the component remains attached to its anchor on scroll."

Reply:
DONE